scripts/systemd/mysqld_pre_systemd.in

```diff
@@ -263,24 +100,14 @@ install_db () {
     # Create initial db and install validate_password plugin
     initfile="$(install_validate_password_sql_file)"
     @libexecdir@/mysqld ${instance:+--defaults-group-suffix=@$instance} --initialize \
-		     --datadir="$datadir" --user=@MYSQLD_USER@ --init-file="$initfile"
+                     --datadir="$datadir" --user=@MYSQLD_USER@ --init-file="$initfile"
     rm -f "$initfile"
 
-<<<<<<< HEAD
     # Generate certs if needed
     if [ -x @bindir@/mysql_ssl_rsa_setup -a ! -e "${datadir}/server-key.pem" ] ; then
         @bindir@/mysql_ssl_rsa_setup --datadir="$datadir" --uid=@MYSQLD_USER@ >/dev/null 2>&1
     fi
     return 0
```

Comment:
mysql_ssl_rsa_setup has been removed by upstream by commit b75bcb61737b. So there is no need of this block. Perhaps a `exit 0` would be enough.

Reply:
fixed